docs/src/prototypes/chatMessages/index.tsx

```diff
@@ -18,11 +17,5 @@ export default () => (
     >
       <ImportantAndMentionMessages />
     </ComponentPrototype>
-    <ComponentPrototype
```

Comment:
Removing this prototype as the reactions with popup is already included in the first one